File: Reforço de Lógica/EX07.php
<?php 
// Recebendo o ponto inicial::
$contador = readline("Iniciar a contagem regressiva de quantos segundos? ");
echo "\n";

// Loop para contar até o número solicitado, de 1 em 1:
for($i = $contador; $i > 0; $i -= 1){
echo "$i\n";
// Espera um segundo:
sleep(1);
}

echo "\n---------------------FIM!---------------------";
?>

File: Reforço de Lógica/Python/EX07.py
# ------------------- Contagem Regressiva ------------------- #

from time import sleep # Importando a biblioteca da função sleep para esperar um segundo entre as linhas de código

contador = int(input("Iniciar a contagem regressiva de quantos segundos? ")) # Pede ao usuário o número inicial da contagem regressiva

print("\n")

while contador > 0: # Enquanto o contador for maior que zero, a contagem regressiva continua
    print(contador) # Mostra o número atual que o contador está
    contador -= 1 # Diminui a variável do contador em -1
    sleep(1) # Espera um segundo antes de seguir o loop

print("\n---------------------FIM!---------------------")